apm-agent-plugins/apm-logging-plugin/apm-jul-plugin/src/test/java/co/elastic/apm/agent/jul/error/JulLoggerErrorCapturingInstrumentationTest.java

```diff
@@ -36,4 +35,12 @@ void captureErrorExceptionWithStringMessage() {
         logger.log(Level.SEVERE, "exception captured", new RuntimeException("some business exception"));
         verifyThatExceptionCaptured(1, "some business exception", RuntimeException.class);
     }
+
+    @Test
+    void captureErrorExceptionWithLogRecord() {
+        LogRecord lr = new LogRecord(Level.WARNING, "exception captured");
```

Comment:
Good test, but it should assert that an error is not reported 🙂 
And an additional with `java.util.logging.Level#SEVERE` would assert for an error being reported